gcp/gae/iam_readwrite.yaml

```diff
@@ -3,6 +3,10 @@ description: Grants Flightcrew's Control Tower VM write access to AppEngine conf
 stage: ALPHA
 # These permissions add to the Read-Only role to deploy new AppEngine versions.
 includedPermissions:
+# Read projects
+- resourcemanager.projects.get
+- resourcemanager.projects.list
```

Comment:
I had changed these files recently so this one is write only, and the others are read only. So we don't need these permissions here since they won't ever be set without the read role also (is that the current logic actually? that you always set read first and then upgrade permissions later?)

Reply:
No, what am I supposed to do here? 🤔 

Like we add a bunch of permissions first, and then we append the permissions?

The commands I'm running are pretty much the same as what are in the `sh` file, which creates separate IAM roles with different sets of permission, and you swap based on whether it's read/write.

It does `add-iam-policy-binding` if that's what you're looking for.

Reply:
It would do the add-binding command on iam_read.yaml, then later do add-binding on iam_write.yaml
So I think this is maybe a TODO for the `upgrade` command where we can upgrade the permissions by attaching the write.yaml

Reply:
Oh, so the install should actually be 

1. First bind the read-only permissions
2. Then bind the read-write permissions if the --write flag is on

Is that right?

Reply:
Yes exactly